### PR TITLE
(fix) [SD-3582] Get rid of the embargo and publish schedulers when executing a "fetch to"

### PR DESCRIPTION
**Pull request by sivakuna-aap**

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/send-item.html [hunk 1]
```diff
@@ -45,7 +44,7 @@ <h3 ng-show="mode !== 'authoring'" translate>Fetch To</h3>
             </ul>
         </div>
         
-        <div sd-toggle-box ng-if="showPublishSchedule()" data-title="Publishing" data-open="true" data-style="dark">
+        <div sd-toggle-box id="publishScheduleTimestamp" ng-if="mode === 'authoring' && showPublishSchedule()" data-title="Publishing" data-open="true" data-style="dark">
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/send-item.html]: This line and line 35 are very long and require significant horizontal scrolling, I would suggest breaking them into multiple lines.

**Comment by plamut** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

**Pull request merged**